src/NzbDrone.Core/Download/Clients/Deluge/Deluge.cs

```diff
@@ -210,7 +210,7 @@ private ValidationFailure TestConnection()
             }
             catch (DownloadClientAuthenticationException ex)
             {
-                _logger.Error(ex, ex.Message);
```

Comment:
Removing these causes the trace to log twice in console and local logs

Reply:
Fixed in the updated version - I've replaced `ex.Message` with a generic string.